logentries/values.yaml

```diff
@@ -0,0 +1,11 @@
+image:
+  name: logentries
```

Comment:
@svenmueller @heshamMassoud 
this property _image.name_ looks confusing: actually it is not an image name, but rather application or container name, see https://github.com/commercetools/k8s-charts/pull/2/files#diff-c65d7160081d712e179b5ec0a5d1c1b0R13

Reply:
@andrii-kovalenko-ct it's not required at all (see my comments in the PR for the crobjob chart)